crates/block-producer/src/errors.rs

```diff
@@ -317,7 +317,7 @@ pub enum StoreError {
     #[error("account Id prefix already exists: {0}")]
     DuplicateAccountIdPrefix(AccountId),
     #[error("gRPC client error")]
-    GrpcClientError(Box<tonic::Status>),
+    GrpcClientError(#[from] Box<tonic::Status>),
```

Comment:
Should the `impl From<Status>` lower down not be removed as well?

Reply:
Need it to map `tonic::Status` to `Box<tonic::Status>` for the purposes of `GrpcClientError(#[from] Box<tonic::Status>)`